components/CopilotTooltip.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useContext} from 'react';
+import {useCopilot} from 'react-native-copilot';
+import {Text, Button, Row, Column} from './../components/ui';
+import {useTranslation} from 'react-i18next';
+import {GlobalContext} from '../shared/GlobalContext';
+import {AuthEvents, selectIsInitialDownload} from '../machines/auth';
+import {useSelector} from '@xstate/react';
+
+export const CopilotTooltip = () => {
+  const {t} = useTranslation('copilot');
+  const {
+    goToNext,
+    goToPrev,
+    stop,
+    currentStep,
+    isFirstStep,
+    isLastStep,
+    totalStepsNumber,
+    copilotEvents,
+  } = useCopilot();
+
+  const {appService} = useContext(GlobalContext);
+  const authService = appService.children.get('auth');
+  const ONBOARDING_DONE = () => authService?.send(AuthEvents.ONBOARDING_DONE());
+  const INITIAL_DOWNLOAD_DONE = () =>
+    authService?.send(AuthEvents.INITIAL_DOWNLOAD_DONE());
+  const isInitialDownloading = useSelector(
+    authService,
+    selectIsInitialDownload,
+  );
+  const handleStop = async () => {
+    stop();
+  };
+
+  const handleNext = () => {
+    void goToNext();
+  };
+
+  const handlePrev = () => {
+    void goToPrev();
+  };
+
+  copilotEvents.on('stop', () => {
+    if (currentStep?.order <= 5) {
+      ONBOARDING_DONE();
+    }
+    if (currentStep?.order === 6) {
```

Comment:
Can we extract constant for the magic numbers 6 & 5 in code?

This way we name the variable about what its about say TOTAL_NUMBER_OF_COPILOT_STEPS and this improves readability. 
https://refactoring.guru/replace-magic-number-with-symbolic-constant